skimage/transform/_geometric.py

```diff
@@ -1447,7 +1448,8 @@ def estimate(self, src, dst, order=2):
         A[:rows, -1] = xd
         A[rows:, -1] = yd
 
-        _, _, V = np.linalg.svd(A)
+        # scipy sparse svd uses less memory
+        _, _, V = svds(A, return_singular_vectors = True)
```

Comment:
```suggestion
        _, _, V = svds(A, return_singular_vectors=True)
```
Are you an R programmer? :wink: It looks like `return_singular_vectors=True)` is the default anyway, so there's no need to specify it, I would say. Can you run the test suite locally?

Reply:
After running the tests I think I figured out the issue. scipy.sparse.linalg.svds() does not guarantee the order of returned singular vectors and hence it may fail the assumption in some cases. I am looking for other alternatives.  If you have any ideas it'll be great.